### DISCUSSION TITLE
Jumping to a season; what am I doing wrong?

### DISCUSSION THREAD

**Discussion by Sarcas666** [pymedusa/Medusa#10139] [Q&A] [answered]
I added a show with 23 seasons and told Medusa to ignore previous shows because I just want to try out season 1. When I select the show, for some reason season 23 is on top (can I reverse this somehow?). Somewhere on the right, I can select 'Jump to season 1', but it doesn't do anything. I can scroll down a bit and set 'row per page' to 'all', but that doesn't stick. I get a few more rows, but have to scroll down a bit, and set it to all again, scroll down a bit, et cetera. Surely I am doing something wrong here? Of have changed a setting?

**Comment by p0psicles** [comment, marked as answer]
You should be able to set pagination to 'all'. Then wait as it will fetch more seasons. And when that finishes you can jump to the season.
But I agree, this isn't working really well.

> can I reverse this somehow?

nope

**Reply by Sarcas666** [reply]
Sad times. But I'll survive. Thx.
